commands/connect.js

```diff
@@ -1,24 +1,35 @@
-const chalk = require('chalk');
-const files = require('../lib/files');
-const git = require('../lib/git');
-const getConfig = require('../lib/config').getConfig;
+const chalk = require("chalk");
+const files = require("../lib/files");
+const git = require("../lib/git");
+const getConfig = require("../lib/config").getConfig;
 
-const connect = (username, project, connection, opts) => {
-  console.log(chalk.cyanBright('Connecting remote origin...'));
-  const githubUsername = username || getConfig('github');
+const connect = (username, project, connection, organization, opts) => {
```

Comment:
5 params is too much. there should ideally be only max 2 params in a function... MAYBE 3 params.

instead of these, just pass in an `options` object. use this pattern: https://rclayton.silvrback.com/easy-class-api-options-with-typescript-and-joi (ignore the Joi library parts)

Reply:
do this after you integrate typescript

Reply:
> do this after you integrate typescript

Monarch I would prefer if anything you want fixed that isn't on this branch be created as an issue this would allow potential community participation.